lib/screen/onboarding/store/on_boarding_store.dart

```diff
@@ -57,6 +59,38 @@ abstract class _OnBoardingStore with Store {
 
   @observable
   bool isLoading = false;
+  static const List<String> scopes = <String>[
+    'email',
+    'https://www.googleapis.com/auth/contacts.readonly',
+  ];
+
+  GoogleSignIn _googleSignIn = GoogleSignIn(
+    serverClientId: BuildConfig.oAuthClientId,
+    scopes: scopes,
+  );
+
+  @action
+  Future<bool> loginWithGoogle() async {
+    try {
+      await _googleSignIn.signIn();
+
+      final googleAuth = await _googleSignIn.currentUser!.authentication;
+      final accessToken = googleAuth.accessToken;
+      final idToken = googleAuth.idToken;
+      print(accessToken);
+      print(idToken);
```

Comment:
```suggestion
```

Let's switch from print methods to `debugPrint` or log from `dart:developer` for production-ready logging. While print is handy for debugging, it's not suitable for deployment due to security considerations. This change will ensure a more secure and production-ready codebase.